docs/howtos/breaking-changes.md

```diff
@@ -0,0 +1,22 @@
+# Breaking changes in application-services code
```

Comment:
How do you feel about adding a reference to this on our PR templates as well? more so for our future selves and other contributors to remember we have to do this